### PR TITLE
[WIP] Issue #4731: add job postings

### PR DESCRIPTION
**Pull request by AnneTee**
Fixes issue [#4731](https://pm.savaslabs.com/issues/4731)

## Summary of changes

1. Builds the basic page layout and styles. This layout can be used with markdown files to create text-based pages similar to blog posts (but much simpler obviously). Basic pages can include a hero image and a CTA region in addition to the page title and body.
2. Adds a basic page template in the `_templates` directory with instructions for creating one
3. Adds `/jobs`. I thought the hero image was well-suited to a general jobs page, but if you don't want to spend the time filling this out yet, I can move it to the marketing director page.
4. Adds `/jobs/director-of-marketing`. 

## Notes

All copy is a draft and will need to be updated. Enjoy the corporate ipsum!

## To test

- [ ] Pull this and run `gulp clean && gulp serve`
- [ ] Check out http://localhost:3000/jobs/ and http://localhost:3000/jobs/director-of-marketing/
- [ ] Ensure things look good at various screen sizes

Feedback welcome!

### Pull request reviewer

As the reviewer, I have verified the following:

- [ ] I have tested the PR locally and/or in a staging environment

### CODE REVIEW

**Comment by chrisarusso** [comment]
First thing, corporate ipsum is _AMAZING_ (read: I say half these things)... more to come.

**Comment by chrisarusso** [comment]
should update "we're hiring" link on /company/, more next week, just didn't want to lose this!

**Review by chrisarusso — CHANGES_REQUESTED** [review]
1) Please review my copy updates
2) I think a header image for the Marketing Director page would be nice as well. I will be linking to it from off-site so many won't see the careers page
3) I still need to update copy on Marketing Director page - just a note

**Review by chrisarusso — APPROVED** [review]

File: _pages/careers.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+excerpt: Career positions to work with the talented Savas Labs team
+hero_region: hero--jobs.html
+cta_heading: Join our team
+cta_paragraph: Interested in working with Savas Labs? Contact us today.
+cta_link: https://savas-labs.breezy.hr/
+cta_link_text: Apply Now
+cta_color: orange
+---
+
+At Savas Labs, we focus on the long-term with both those we partner with as well as our internal team. We have a core set of [values](company/mission-and-values/) that unite us, yet celebrate that which makes us each unique. We work hard, have fun, and strive for healthy balance. Some of our current employment benefits are:
```
Comment (AnneTee) [inline _pages/careers.md]: @chrisarusso You need a leading / in the link to the mission & values page

File: _pages/careers.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+ - generous PTO
```
Comment (AnneTee) [inline _pages/careers.md]: Let's capitalize the first letter of each bullet point

**Pull request merged**